tedana/model/fit.py

```diff
@@ -57,33 +58,38 @@ def fitmodels_direct(catd, mmix, mask, t2s, t2s_full, tes, combmode, ref_img,
     Returns
     -------
     seldict : dict
-    comptab : (N x 5) :obj:`numpy.ndarray`
+    comptab : (N x 5) :obj:`pandas.DataFrame`
         Array with columns denoting (1) index of component, (2) Kappa score of
         component, (3) Rho score of component, (4) variance explained by
-        component, and (5) normalized variance explained bycomponent
+        component, and (5) normalized variance explained by component
     betas : :obj:`numpy.ndarray`
     mmix_new : :obj:`numpy.ndarray`
     """
     if not (catd.shape[0] == t2s.shape[0] == t2s_full.shape[0] == mask.shape[0]):
         raise ValueError('First dimensions (number of samples) of catd ({0}), '
                          't2s ({1}), and mask ({2}) do not '
-                         'match'.format(catd.shape[0], t2s.shape[0], mask.shape[0]))
+                         'match'.format(catd.shape[0], t2s.shape[0],
+                                        mask.shape[0]))
     elif catd.shape[1] != len(tes):
-        raise ValueError('Second dimension of catd ({0}) does not match number '
-                         'of echoes provided (tes; {1})'.format(catd.shape[1], len(tes)))
+        raise ValueError('Second dimension of catd ({0}) does not match '
+                         'number of echoes provided (tes; '
+                         '{1})'.format(catd.shape[1], len(tes)))
     elif catd.shape[2] != mmix.shape[0]:
         raise ValueError('Third dimension (number of volumes) of catd ({0}) '
                          'does not match first dimension of '
                          'mmix ({1})'.format(catd.shape[2], mmix.shape[0]))
     elif t2s.shape != t2s_full.shape:
         raise ValueError('Shape of t2s array {0} does not match shape of '
-                         't2s_full array {1}'.format(t2s.shape, t2s_full.shape))
+                         't2s_full array {1}'.format(t2s.shape,
+                                                     t2s_full.shape))
     elif t2s.ndim == 2:
         if catd.shape[2] != t2s.shape[1]:
-            raise ValueError('Third dimension (number of volumes) of catd ({0}) '
-                             'does not match second dimension of '
+            raise ValueError('Third dimension (number of volumes) of catd '
+                             '({0}) does not match second dimension of '
                              't2s ({1})'.format(catd.shape[2], t2s.shape[1]))
 
+    mask = t2s != 0  # Override mask because problems
```

Comment:
This is a hacky workaround to deal with the fact that many arrays (e.g., `F_R2_clmaps` and `Br_R2_clmaps`) are differently sized.